src/components/Timeinfo/Timeinfo.tsx

```diff
@@ -10,13 +10,13 @@ export default function Timeinfo(props: INewDate) {
 
   useEffect(() => {
     const intervalId = setInterval(() => {
-      setTime(new Date());
+      setTime(time);
     }, 1000);
 
     return () => {
       clearInterval(intervalId);
     };
-  }, []);
+  }, [time]);
```

Comment:
No need in this code, let's remove for now

Reply:
Done)